app/controllers/organisations/detail.js

```diff
@@ -17,32 +16,25 @@ export default Ember.Controller.extend(SearchOptionMixin, AsyncMixin, {
     return !this.get("country.id");
   }),
 
-  isInValidWebsite: Ember.computed("model.website", function() {
-    const websiteRegEx = new RegExp(regex.WEBSITE_REGEX);
-
-    return (
-      this.get("model.website") &&
-      !this.get("model.website").match(websiteRegEx)
-    );
-  }),
-
   actions: {
     /**
      * Updates the organisation if
      *      nameEn is present
      *      country is present
      *      type is present
-     *      website has a valid format iff its present
      */
-    updateOrganisation() {
-      if (
-        !this.get("isInValidNameEn") &&
-        !this.get("isInValidCountry") &&
-        !this.get("isInValidWebsite")
-      ) {
-        this.send("update");
-      } else {
-        this.set("showError", true);
+    updateOrganisation(e) {
+      let isValid = true;
+      switch (e.target.name) {
+        case "name_en":
+          isValid = !Boolean(this.get("isInValidNameEn"));
+          break;
+      }
```

Comment:
```suggestion
      if (e.target.name === "name_en") {
          isValid = !Boolean(this.get("isInValidNameEn"));
      }
```

Reply:
I have purposefully kept it this way, as it might get ugly in case there are multiple fields to be validated